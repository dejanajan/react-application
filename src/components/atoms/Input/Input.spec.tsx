import React from 'react';
import InputField from './Input.component'
import renderer from 'react-test-renderer';

test('Input Field renders correctly', () => {
  const inputField = renderer
    .create( <InputField onChange={() => null} value="Repo"/>)
    .toJSON();

  expect(inputField).toMatchSnapshot();
});