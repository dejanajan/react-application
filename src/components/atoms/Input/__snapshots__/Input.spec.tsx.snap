// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Input Field renders correctly 1`] = `
<div
  className="sc-AxjAm jGzjec"
>
  <label
    htmlFor="filterRepos"
  >
    Filter Repos
  </label>
  <br />
  <input
    id="filterRepos"
    onChange={[Function]}
    type="text"
    value="Repo"
  />
</div>
`;
